CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 3.1.1 (March 16, 2022)
+
+NOTES:
+
+* Updated [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs) to `v0.7.0`:
+  this improves generated documentation, with attributes now correctly formatted as `code`
+  and provided with anchors.
+* Updated use of internal tool to include missing release asset.
```

Comment:
This was not practitioner effecting so it does not need to be included in the CHANGELOG.

```suggestion
```